storm-core/src/jvm/backtype/storm/Config.java

```diff
@@ -247,6 +253,7 @@
      *
      * Defaults to false.
      */
+    @Deprecated
```

Comment:
This should actually not be deprecated.  It says zmq, but it is not ZMQ.  It should probably be renamed, but that is a follow on JIRA.